ROS/osr/src/roboclaw_wrapper.py

```diff
@@ -386,46 +385,50 @@ def velocity2qpps(self, velocity, ticks_per_rev, gear_ratio):
         """
         return int(velocity * gear_ratio * ticks_per_rev / (2 * math.pi))
 
-    def getBattery(self):
-        return self.rc.ReadMainBatteryVoltage(self.address[0])[1]
+    def read_battery(self):
+        """Read battery voltage from one of the roboclaws as a proxy for all roboclaws"""
+        # roboclaw reports value in 10ths of a Volt
+        return self.rc.ReadMainBatteryVoltage(self.address[0])[1] / 10.0
 
-    def getTemp(self):
+    def read_temperatures(self):
         temp = [None] * 5
         for i in range(5):
-            temp[i] = self.rc.ReadTemp(self.address[i])[1]
+            # reported by roboclaw in 10ths of a Celsius
+            temp[i] = self.rc.ReadTemp(self.address[i])[1] / 10.0
+        
         return temp
 
-    def getCurrents(self):
+    def read_currents(self):
         currents = [None] * 10
         for i in range(5):
             currs = self.rc.ReadCurrents(self.address[i])
-            currents[2*i] = currs[1]
-            currents[(2*i) + 1] = currs[2]
+            # reported by roboclaw in 10ths of an Ampere
+            currents[2*i] = currs[1] / 10.0
+            currents[(2*i) + 1] = currs[2] / 10.0
```

Comment:
```suggestion
            currents[2*i] = currs[1] / 100.0
            currents[(2*i) + 1] = currs[2] / 100.0
```

Reply:
@gfosmire just an FYI this is a cool feature of GitHub where you can submit suggestions to code so the PR opener can merge in those changes through GitHub. Select a couple of lines in review mode and hit the left-most icon 'Suggestion' and when you make changes, it'll show up like the above